File: ts/src/pull-with-push-override/run/run-baz-override.ts
/*
 * Copyright © 2022 Paulo Villela. All rights reserved.
 * Use of this source code is governed by the MIT license
 * that can be found in the LICENSE file.
 */

import { bazCfgSrc, baz } from "../fs/baz";
import { thunkOf } from "../../arch/util/functions";

bazCfgSrc.set(thunkOf({ w: "baz" }))

baz()

File: src/main/kotlin/pullcfgwithoverride/run/runBazOverride.kt
/*
 *  Copyright © 2022 Paulo Villela. All rights reserved.
 *  Use of this source code is governed by the MIT license
 *  that can be found in the LICENSE file.
 */

package pullcfgwithoverride.run

import pullcfgwithoverride.fs.BazCfgInfo
import pullcfgwithoverride.fs.baz
import pullcfgwithoverride.fs.bazCfgSrc

fun main() {
    bazCfgSrc = { BazCfgInfo("baz") }

    baz()
}
